electrumsv/wallet_database/functions.py

```diff
@@ -2310,3 +2308,41 @@ async def update_transaction_proof_async(self, tx_hash: bytes, block_height: int
             block_position: int, proof: TxProof) -> None:
         await self._db_context.run_in_thread_async(_update_transaction_proof, tx_hash,
             block_height, block_position, proof)
+
+
+def create_mapi_broadcast_callbacks(db_context: DatabaseContext,
+        rows: Iterable[MAPIBroadcastCallbackRow]) -> concurrent.futures.Future[None]:
+    sql = f"""
+        INSERT INTO MAPIBroadcastCallbacks
+        (tx_hash, peer_channel_id, broadcast_date, encrypted_private_key, server_id, status_flags)
+        VALUES (?, ?, ?, ?, ?, ?)"""
+
+    def _write(db: sqlite3.Connection) -> None:
+        db.executemany(sql, rows)
+    return db_context.post_to_thread(_write)
+
+
+@replace_db_context_with_connection
+def read_mapi_broadcast_callbacks(db: sqlite3.Connection) -> List[MAPIBroadcastCallbackRow]:
+    sql = f"""
+        SELECT *
```

Comment:
Replace * with column names so ordering of column values is explicit and anyone reading this can see what is returned.